src/baldr/tz_alt.cpp

```diff
@@ -0,0 +1,4025 @@
+// The MIT License (MIT)
```

Comment:
so this is the original `tz.cpp` but with a couple methods added to load tz data from strings. the diff is quite small. in the future we should write a target in cmake to apply a patch to `tz.cpp` and compile that.